boards/riscv32/arty_litex_vexriscv/Kconfig.board

```diff
@@ -0,0 +1,4 @@
+config BOARD_ARTY_LITEX_VEXRISCV
+	bool "Digilent Arty dev board with LiteX/VexRiscV CPU"
+	select HAS_DTS
```

Comment:
The select HAS_DTS shouldn't be needed.  Should already be set for any RISC-V 32 port.

Reply:
You are right, we checked - it's not needed and will be removed.

Reply:
It's done.